rdmd.d

```diff
@@ -854,12 +835,11 @@ string makeEvalFile(string todo)
     enum lifetimeInHours = 24;
     auto cutoff = Clock.currTime() - dur!"hours"(lifetimeInHours);
     yap("dirEntries ", pathname);
-    foreach (DirEntry d; dirEntries(pathname, SpanMode.shallow))
+    foreach (d; std.file.dirEntries(pathname, std.file.SpanMode.shallow))
```

Comment:
This is odd: why not create a similar wrapper function here that handles the `yap` under the hood?  Why preserve a manual `yap` call in the code as is done here?

Reply:
seemed unnecessary here.  this function is only called once and the wrapper function would be larger than then other functions.